packages/extension-chakra-storefront/src/pages/product-detail/index.jsx

```diff
@@ -450,18 +450,7 @@ const ProductDetail = () => {
             layerStyle="page"
             data-testid="product-details-page"
         >
-            <Helmet>
-                <title>{product?.pageTitle}</title>
-                {product?.pageMetaTags?.length > 0 &&
-                    product.pageMetaTags.map(({id, value}) => (
-                        <meta name={id} content={value} key={id} />
-                    ))}
-                {/* Fallback for description if not included in pageMetaTags */}
-                {!product?.pageMetaTags?.some((tag) => tag.id === 'description') &&
-                    product?.pageDescription && (
-                        <meta name="description" content={product.pageDescription} />
-                    )}
-            </Helmet>
```

Comment:
Moving the metadata logic out from the index file in order to keep the index files smaller.